src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/ClientConfiguration.java

```diff
@@ -0,0 +1,39 @@
+#set($symbol_pound='#')
+#set($symbol_dollar='$')
+#set($symbol_escape='\' )
+package ${package}.${artifactId}.configuration;
+
+import com.rebuy.sdk.customer.CustomerClient;
+import ${package}.${artifactId}.configuration.settings.*;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.client.ClientProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+@Profile("!testing")
+public class ClientConfiguration
+{
+    @Bean
+    public CustomerClient customerClient(
```

Comment:
I would let it there, if we have consul rolled out in all services, you will only need to initialize the client without any configuration. Then we can remove it.
A example always helps and most new silos should need that client.